code/datums/mutations/adaptation.dm

```diff
@@ -1,53 +1,25 @@
+/datum/mutation/human/Space_adaptation
+	name = "Space Adaptation"
+	desc = "A strange mutation that renders the host immune to damage from extreme temperatures and pressure."
 	quality = POSITIVE
 	difficulty = 16
 	text_gain_indication = "<span class='notice'>Your body feels warm!</span>"
-	instability = 25
-	conflicts = list(/datum/mutation/human/pressure_adaptation)
+	instability = 35
 
-/datum/mutation/human/temperature_adaptation/New(class_ = MUT_OTHER, timer, datum/mutation/human/copymut)
+/datum/mutation/human/Space_adaptation/New(class_ = MUT_OTHER, timer, datum/mutation/human/copymut)
 	..()
 	if(!(type in visual_indicators))
 		visual_indicators[type] = list(mutable_appearance('icons/effects/genetics.dmi', "fire", -MUTATIONS_LAYER))
 
-/datum/mutation/human/temperature_adaptation/get_visual_indicator()
+/datum/mutation/human/Space_adaptation/get_visual_indicator()
 	return visual_indicators[type][1]
 
-/datum/mutation/human/temperature_adaptation/on_acquiring(mob/living/carbon/human/owner)
+/datum/mutation/human/Space_adaptation/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
-	owner.add_traits(list(TRAIT_RESISTCOLD, TRAIT_RESISTHEAT), GENETIC_MUTATION)
+	owner.add_traits(list(TRAIT_RESISTCOLD, TRAIT_RESISTHEAT, TRAIT_RESISTLOWPRESSURE, TRAIT_RESISTHIGHPRESSURE), GENETIC_MUTATION)
 
-/datum/mutation/human/temperature_adaptation/on_losing(mob/living/carbon/human/owner)
+/datum/mutation/human/Space_adaptation/on_losing(mob/living/carbon/human/owner)
 	if(..())
 		return
-	owner.remove_traits(list(TRAIT_RESISTCOLD, TRAIT_RESISTHEAT), GENETIC_MUTATION)
-
-/datum/mutation/human/pressure_adaptation
-	name = "Pressure Adaptation"
-	desc = "A strange mutation that renders the host immune to damage from both low and high pressure environments. Does not protect from temperature, including the cold of space."
-	quality = POSITIVE
-	difficulty = 16
-	text_gain_indication = "<span class='notice'>Your body feels numb!</span>"
-	instability = 25
-	conflicts = list(/datum/mutation/human/temperature_adaptation)
-
-/datum/mutation/human/pressure_adaptation/New(class_ = MUT_OTHER, timer, datum/mutation/human/copymut)
-	..()
-	if(!(type in visual_indicators))
-		visual_indicators[type] = list(mutable_appearance('icons/effects/genetics.dmi', "pressure", -MUTATIONS_LAYER))
-
-/datum/mutation/human/pressure_adaptation/get_visual_indicator()
-	return visual_indicators[type][1]
-
-/datum/mutation/human/pressure_adaptation/on_acquiring(mob/living/carbon/human/owner)
-	if(..())
-		return
-	owner.add_traits(list(TRAIT_RESISTLOWPRESSURE, TRAIT_RESISTHIGHPRESSURE), GENETIC_MUTATION)
-
-/datum/mutation/human/pressure_adaptation/on_losing(mob/living/carbon/human/owner)
-	if(..())
-		return
-	owner.remove_traits(list(TRAIT_RESISTLOWPRESSURE, TRAIT_RESISTHIGHPRESSURE), GENETIC_MUTATION)
+	owner.remove_traits(list(TRAIT_RESISTCOLD, TRAIT_RESISTHEAT, TRAIT_RESISTLOWPRESSURE, TRAIT_RESISTHIGHPRESSURE), GENETIC_MUTATION)
```

Comment:
this is no longer space adapt as it gives high pressure resistance, its literally just immune to all atmospheric negative effect